torchaudio/csrc/build_rir.cpp

```diff
@@ -0,0 +1,187 @@
+#include <math.h>
+#include <torch/script.h>
+#include <torch/torch.h>
+using namespace torch::indexing;
+
+namespace torchaudio {
+namespace rir {
```

Comment:
Since this file does not have corresponding header file, can you add anonymous namespace so that it is not referable from other C++ source?